app/routes/payment-requests.js

```diff
@@ -0,0 +1,51 @@
+const { post } = require('../api')
+const Joi = require('joi')
+const { schemeAdmin } = require('../auth/permissions')
+
+module.exports = [{
+  method: 'GET',
+  path: '/payment-request/reset',
+  options: {
+    auth: { scope: [schemeAdmin] },
+    handler: async (request, h) => {
+      return h.view('reset-payment-request')
+    }
+  }
+},
+{
+  method: 'POST',
+  path: '/payment-request/reset',
+  options: {
+    auth: { scope: [schemeAdmin] },
+    validate: {
+      payload: Joi.object({
+        invoiceNumber: Joi.string().required().error(errors => {
+          errors.forEach(err => { err.message = 'Enter a valid invoice number' })
+          return errors
+        })
```

Comment:
Should be pulled out into its own invoiceNumber schema component and then added to the routes own schema file like in `ffc-pay-request-editor`: https://github.com/DEFRA/ffc-pay-request-editor/tree/main/app/routes/schemas

This also adds some coals to the fire re: 1 route path (no sub-routes), i.e. payment-request (GET, POST, DELETE, etc no payment-request/reset GET, POST, DELETE, etc) to 1 route file

Reply:
It could, but as it's only used in one place, I'm not sure it adds any value, other than meaning a reader has to look in two places.  But I think it does make the routes easier to read by moving it.

I'll move it.